### ISSUE TITLE
[Local API] Error 400 Unknown object. (UNSPECIFIED_ERROR)

### ISSUE THREAD

**Issue by dubocr** [dubocr/homebridge-tahoma#405]
Using the local API, I get Error 400 Unknown object. (UNSPECIFIED_ERROR)
![](https://user-images.githubusercontent.com/490610/190929288-ea29f451-b2f8-44ae-9de6-e73f816ea72f.png)

**Comment by dubocr** [comment]
@qelanhari you said the error disappeared suddenly ?

**Comment by qelanhari** [comment]
Nope, it was just that i reduced back the refreshperiod. Still got it when it does refresh.

**Comment by dubocr** [comment]
@IngmarStein @qelanhari

Full device refresh is not yet available on Local API : [https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/26](https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/26).

Until supported, the error could be ignored.

**Comment by qelanhari** [comment]
Can we use the setup endpoint instead ? :-p

**Comment by IngmarStein** [comment]
What's the consequence of this API not being available, i.e. what feature requires full device refresh?

**Comment by dubocr** [comment]
This endpoint request all devices to refresh their states.

This is used for devices that were operated locally (manually or by remote control) but don't `report` their states to the controller (bridge).

With io-homecontrol, devices could be actuators or sensors (or both). actuators receive radio orders but don't notify their states. sensors notify their states.

In the case a device (eg. a garage door) is `actuator` only, if it is operated locally (eg. with radio remote control) the `OpenClosed` state of the device is updated internally but not reported (notified) to the controller (the bridge).

So we have to periodically `refresh` the device states to make sure bridge (and so homebridge) is up to date.

Because of this limitation on `actuators`, manufacturers generally implements new devices as `actuator + sensor` to let them notify their states. But it is not the case for all devices, particularly old one.

Setup endpoint just retrieve the full configuration from the bridge. But as explained above, even the bridge is not up to date in this case. (getting the fresh data from the bridge is done with the `fetch` request made every 10 seconds).

**Comment by PrzemyslawKlys** [comment]
Have the same issue:

![image](https://user-images.githubusercontent.com/15063294/201645237-fca272d6-9e17-4ff0-a4f2-ee5cf615cbb9.png)

**Comment by MirkoKas** [comment]
> @IngmarStein @qelanhari
> 
> Full device refresh is not yet available on Local API : [Somfy-Developer/Somfy-TaHoma-Developer-Mode#26](https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/26).
> 
> Until supported, the error could be ignored.

@dubocr I have the same „issue“ or better annoyance. I understand the limitations of the local API here but would it be possible to hide that message in the homebridge logs for local API configuration?

**Comment by piejanssens** [comment]
Because it might take a while before this is supported by the local API, maybe catch and report accordingly in the console until then?

**Comment by FortuneV13** [comment]
Same issue here. Would be nice to hide this from the Logs until supported.

**Comment by Tontonpat** [comment]
Bonjour Romain, c’est vrai que ce serait bien de masquer ce message qui pollue les logs

**Comment by Tontonpat** [comment]
Bonjour @dubocr est ce possible du supprimer ce message qui pollue les logs?
Merci

**Comment by Tontonpat** [comment]
@dubocr up stp
